packit/upstream.py

```diff
@@ -400,17 +400,26 @@ def bump_spec(self, version: str = None, changelog_entry: str = None):
         cmd.append(str(self.absolute_specfile_path))
         run_command(cmd)
 
-    def set_spec_version(self, version: str, changelog_entry: str):
+    def set_spec_version(
+        self, version: str = None, release: str = None, changelog_entry: str = None
+    ):
         """
         Set version in spec and add a changelog_entry.
 
         :param version: new version
         :param changelog_entry: accompanying changelog entry
         """
         try:
-            # also this code adds 3 rpmbuild dirs into the upstream repo,
-            # we should ask rebase-helper not to do that
-            self.specfile.set_version(version=version)
+            if version:
+                # also this code adds 3 rpmbuild dirs into the upstream repo,
+                # we should ask rebase-helper not to do that
```

Comment:
rpmbuild does that, we could have them in /tmp and then maybe clean them?

Reply:
but honestly, everyone who does rpm packaging is probably aware of this

Reply:
It's your comment. I only moved that...;-)

Reply:
lol